app/frontend/src/components/admin/Developer.vue

```diff
@@ -62,7 +62,9 @@ export default {
   },
   data() {
     return {
+      consoleErrMsg: this.$t('trans.developer.notificationConsErr'),
       apiRes: '',
+      message: this.$t('trans.developer.notificationMsg'),
```

Comment:
Same as earlier, why declare a variable for this translation instead of just using it directly where it's needed? If functionality is extended, then you would end up with two variables. `message` and `message2`.

Reply:
I did declare that variable. It was already declared, I just did the translation

Reply:
We should remove this variable anyways and use the direct translation where it's needed.